### PR TITLE
Shotgun Overhaul 

### PR DESCRIPTION
**Pull request by AmoryBlaine**
The intent of this PR is to repurpose the various shotguns into a role of their own as primarily functioning to focus knockback and stun damage to xenomorphs, rather than function as outright killers. The intended goal of this is to fit shotguns better into the gameplay of PvE. Where hordes can easily overwhelm you even with heavy firepower, the shotgun will be able to buy you a moment of relief by knocking back advancing xenomorphs allowing your teammates to sustain accurate fire and maintain your position.
- **_Super Buck is moved to GM-spawn only, with standard buck and slugs taking its place._**
- Buckshot has been made accurate at a distance and provides spread of knockback pellets that will apply their effect within a range of 4 tiles while doing less damage than previously. They provide no armor penetration.
- Slugs are made accurate to 8 tiles and do a high degree of damage, and can stun individual xenomorphs for a longer period of time than buckshot to balance with its lack of spread. It has high armor penetration.
- The Mk221 is given an increased fire rate, removing it's intense delay between shots. **_The internal magazine has been dropped to 6 in the tube, one in the chamber._**
- The M37 has been given a buff to firing, with there no longer being a delay after pumping. **_The internal magazine has dropped to four shells in the tube, and one chambered._** Also it's now known as the Ithaca 37, customized.
- Undecided for incendiary and flechette ammo changes
- UPP Type 23 has had its fire delay lowered and 8 gauge is now Super Buck-lite. Handle with caution or you will instantly fuck your friends out of most of their health.
- XM38 is has an increased shell count to 7 in the tube, one in the chamber and loses buffs. It's essentially a full auto ithaca.
- HG is left the way it is. It's an Ithaca but with two seperate tubes of 4 shells each.
- Shotguns spawn unloaded, as does the M79 cause I thought why not. 
- Shotguns and M79 get repalettes for on-mobs

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
uhhh this kinda directly conflicts w/ the use of the M4RA. not a huge fan :T

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/datums/ammo/bullet/shotgun.dm [hunk 1]
```diff
@@ -209,24 +243,53 @@
...
+		var/mob/living/carbon/xenomorph/target = living_mob
+		to_chat(target, SPAN_XENODANGER("You are shaken and slowed by the sudden impact!"))
+		target.KnockDown(5)
+		target.Stun(5)
+		target.Slow(8)
+	else
+		if(!isyautja(living_mob)) //Not predators.
+			living_mob.KnockDown(2)
+			living_mob.Stun(2)
+			living_mob.Slow(2)
```
Comment (Doubleumc) [inline code/datums/ammo/bullet/shotgun.dm:268]:
```suggestion
			living_mob.KnockDown(2)
			living_mob.Stun(2)
```
Slowed for the same duration you can't move, therefore slowing never matters.

File: code/datums/ammo/bullet/shotgun.dm [hunk 2]
```diff
@@ -209,24 +243,53 @@
...
```
Comment (Doubleumc) [inline code/datums/ammo/bullet/shotgun.dm:289]:
```suggestion
			living_mob.KnockDown(2)
			living_mob.Stun(2)
```
Slowed for the same duration you can't move, therefore slowing never matters.

**Review by Doubleumc — CHANGES_REQUESTED** [review]
Either make the indicated slowdowns long enough to matter or remove them. Otherwise looks good.

File: code/datums/ammo/bullet/shotgun.dm (hunk 1 above)
Comment (AmoryBlaine) [reply code/datums/ammo/bullet/shotgun.dm:268]: Yeah forgot to fill in () when I copypasted it

**Review by Doubleumc — APPROVED** [review]

**Pull request merged**